collect_app/src/main/java/org/odk/collect/android/http/injection/HttpComponent.java

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.http.injection;
+
+import org.odk.collect.android.http.CollectServerClient;
+import org.odk.collect.android.preferences.ServerPreferencesFragment;
+import org.odk.collect.android.receivers.NetworkReceiver;
+import org.odk.collect.android.tasks.InstanceServerUploader;
+import org.odk.collect.android.utilities.DownloadFormListUtils;
+import org.odk.collect.android.utilities.FormDownloader;
+
+import javax.inject.Singleton;
+
+import dagger.Component;
+
+@Component(modules = HttpInterfaceModule.class)
```

Comment:
We should look into a way to either use this or remove it! Having `Http` modules are really good though. What's the reason why it isn't being used?